db/structure.sql

```diff
@@ -261,13 +288,13 @@ CREATE TABLE public.collections (
     access character varying,
     required_license character varying,
     default_license character varying,
-    email_when_participants_changed boolean DEFAULT true NOT NULL,
```

Comment:
are there settings to be changed so there is less structure.sql churn?  I think this got added by you on an earlier PR, but I'm not certain.